src/config/configTypes.ts

```diff
@@ -32,6 +33,7 @@ export type RepomixConfigDefault = RepomixConfigBase & {
     removeEmptyLines: boolean;
     topFilesLength: number;
     showLineNumbers: boolean;
+    copyToClipboard?: boolean;
```

Comment:
Could you make the following changes to the type definitions:

1. RepomixConfigDefault
```diff
- copyToClipboard?: boolean;
+ copyToClipboard: boolean;
```

2. Then add a default value in `src/config/defaultConfig.ts`:
```typescript
copyToClipboard: false,
```

This makes it consistent with how we handle other boolean config options in the codebase. It also ensures type safety without needing to check if the value is undefined.

Side note: We're planning to migrate to Zod in the future, so this somewhat cumbersome process is temporary...

Reply:
This makes complete sense. Thank you for your clarification!